src/tools/pkcs11-tool.c

```diff
@@ -300,6 +301,7 @@ static const struct option options[] = {
 	{ "generate-random",	1, NULL,		OPT_GENERATE_RANDOM },
 	{ "allow-sw",		0, NULL,		OPT_ALLOW_SW },
 	{ "iv",			1, NULL,		OPT_IV },
+	{ "mac-general-param",	1, NULL, 		OPT_MAC_GEN_PARAM},
```

Comment:
last things:
 * The new cli switches need to be added to the manual pages
 * there are some slight formatting suggestions. You do not need to reformat all of the tables, but the new code should follow the formatting if possible.